modules/core/CMakeLists.txt

```diff
@@ -60,6 +60,23 @@ if(CV_TRACE AND HAVE_ITT)
   add_definitions(-DOPENCV_WITH_ITT=1)
 endif()
 
+# https://github.com/opencv/opencv/issues/24145
+ocv_check_environment_variables(OPENCV_IPP_MEAN)
```

Comment:
Lets guard this block by `if(HAVE_IPP)`

Reply:
I agree with you, thank you! 
And I tried with raspi(arm32, not intel), `OPENCV_IPP_GAUSSIAN_BLUR` is shown in cmake gui. So I added fot it also.